src/analysis/lattices/powerset.h

```diff
@@ -89,6 +84,11 @@ class FiniteIntPowersetLattice {
 
   // Returns an instance of the bottom lattice element.
   Element getBottom() const noexcept;
+
+  // Calculates the LUB of this element with some other element and sets
+  // this element to the LUB in place. Returns true if this element before
+  // this method call was different than the LUB.
```

Comment:
Maybe comment on how the name is a shorthand, as explained in the PR description?

Reply:
+1 yes please